src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -254,7 +266,8 @@ private Result executeCmd(String cmd) throws IOException, InterruptedException {
                     .stderr(stderr)
                     .cmdAsSingleString(cmd)
                     .start().join() == 0;
-            String stdoutStr = stdout.toString();
+            String stdoutStr = baos.toString(Computer.currentComputer().getDefaultCharset().name());
+            IOUtils.closeQuietly(baos);
```

Comment:
<code>closeQuietly()</code> is designed for <code>finally()</code> blocks. Here it makes sense to propagate the exception to caller methods, because there's no need in the interface update